esdm/esdm_es_sched.c

```diff
@@ -208,7 +208,7 @@ static int esdm_sched_seed_monitor(void)
 	if (!esdm_config_es_sched_entropy_rate())
 		return 0;
 
-	if (ent >= esdm_config_es_sched_entropy_rate()) {
+	if (ent >= esdm_config_es_sched_entropy_rate() + esdm_compress_osr()) {
```

Comment:
Why is this? The schedule ES kernel module already reduces the entropy rate by this.